SPECS/mstflint/mstflint.spec

```diff
@@ -23,15 +25,29 @@ BuildRequires:  libxml2-devel
 BuildRequires:  make
 BuildRequires:  openssl-devel
 BuildRequires:  zlib-devel
+Requires:       kernel
```

Comment:
The `Requires` is actually a per-package information unlike `BuildRequires`, which give you build-time dependencies for the whole spec. What that means is that now you've defined a run-time requirement for `msft` package, while we're interested in the `msft-kernel` one. You'd need to move the `Requires` under `%package kernel`.

Btw. the published `msft-kernel` package will end up being built from the `msft-signed.spec` and not from this one, so the run-time requirements are not influencing anyone in this special case. Keeping the `Requires` in both specs, however, is probably going to be less confusing, so I'd still leave them.

And please also specify the exact kernel version the modules needs. Same idea as with the `BuildRequires: kernel-devel` comment above.